src/hooks/super-admin/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { filter, find, findIndex } from "lodash";
+import { User, UserObjectForSuperAdmin } from "../../utilily/types";
+import { Role } from "../../utilily/enums";
+
+export const showAdminName = (
+  connectedAdmin: User["adminID"],
+  allAdminsAnsSuperAdmin: User[] | undefined,
+) => {
+  const admin = find(allAdminsAnsSuperAdmin, { id: connectedAdmin });
```

Comment:
Why are you using lodash's find method here?
No need to use external packages when JavaScript has it's own built in.

Reply:
it looks cleaner. 
on js it would look like this.      allAdminsAnsSuperAdmin.find((element) => element. id ===  connectedAdmin);

Reply:
> it looks cleaner. on js it would look like this. allAdminsAnsSuperAdmin.find((element) => element. id === connectedAdmin);

I've read that lodash is a pretty heavy package.
Plus in my opinion it's unnecessary to use another package that does the exact same as the built in one.

Reply:
That lodash is a pretty heavy package - that I didn't know. 
I checked, and it's fine to import the function directly  like this " import filter from 'lodash/filter'"
in this case, I agree that it's better to use js function even those the lodash looks cleaner.